seacrowd/sea_datasets/cebuaner/cebuaner.py

```diff
@@ -0,0 +1,189 @@
+from pathlib import Path
+from typing import Dict, Iterable, List, Tuple
+
+import datasets
+from datasets.download.download_manager import DownloadManager
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = r"""
+@misc{pilar2023cebuaner,
+    title={CebuaNER: A New Baseline Cebuano Named Entity Recognition Model},
+    author={Ma. Beatrice Emanuela Pilar and Ellyza Mari Papas and Mary Loise Buenaventura and Dane Dedoroy and Myron Darrel Montefalcon and Jay Rhald Padilla and Lany Maceda and Mideth Abisado and Joseph Marvin Imperial},
+    year={2023},
+    eprint={2310.00679},
+    archivePrefix={arXiv},
+    primaryClass={cs.CL}
+}
+"""
+
+_LOCAL = False
+_LANGUAGES = ["ceb"]
+_DATASETNAME = "cebuaner"
+_DESCRIPTION = """\
+The CebuaNER dataset contains 4000+ news articles that have been tagged by
+native speakers of Cebuano usin gthe BIO encoding schema for the named entity
+recognition (NER) task.
+"""
+
+_HOMEPAGE = "https://github.com/mebzmoren/CebuaNER"
+_LICENSE = Licenses.CC_BY_NC_SA_4_0.value
+_URLS = {
```

Comment:
is it better to refer this as `Annotator 1` and `Annotator 2` rather than 0 and 1? SInce this doesn't refer to any list-indexing (and better + more intuitive naming)

Reply:
Done!